Ghidra/Framework/SoftwareModeling/src/main/java/ghidra/pcode/loadimage/LoadImage.java

```diff
@@ -23,18 +23,19 @@
 // using 1 of possibly many different methods behind the scenes
 public interface LoadImage {
 
-    /**
-     * 
-     * @param ptr
-     * @param size
-     * @param addr
-     * @param bufOffset
-     * @param generateInitializedMask if true the function should return an initialized bit mask
-     * or null if all loaded bytes were known.  If true, uninitialized memory reads should only be 
-     * reflected in the mask and should not be reported via the memory fault handler.
-     * @return initialized bit mask or null (see generateInitializedMask parameter)
-     * @see MemoryPage
-     */
+// TODO this doesn't appear to be used.
+//    /**
+//     * 
+//     * @param ptr
+//     * @param size
+//     * @param addr
+//     * @param bufOffset
+//     * @param generateInitializedMask if true the function should return an initialized bit mask
+//     * or null if all loaded bytes were known.  If true, uninitialized memory reads should only be 
+//     * reflected in the mask and should not be reported via the memory fault handler.
+//     * @return initialized bit mask or null (see generateInitializedMask parameter)
+//     * @see MemoryPage
+//     */
```

Comment:
I'm not sure why this class even exists. Its not used anywhere so I didn't bother actually fixing the documentation and just commented it out to remove the javadoc errors.